.github/ISSUE_TEMPLATE.md

```diff
@@ -1,26 +1,38 @@
+<!-- 
+Note:
+
 Please check the existing issues to make sure you're not duplicating a report.
 
 For bug reports, please provide the following information:
 
-In a terminal, run `zmanage info` and paste the output below:
+In a terminal, run `zmanage info` and paste the output below: -->
 
+#### Details:
 ```
 #Paste the output here
 ```
 
-- gist of .zimrc:
-- gist of .zshrc:
+##### Gist of .zimrc:
+  ```
+  #Paste the output here
+  ```
+##### Gist of .zshrc:
+  ```
+  #Paste the output here
+  ```
+
+#### Description
 
-Description
------------
+#### Do you want to request a *feature* or report a *bug*?
+*Bug*
```

Comment:
Maybe we don't need this, since the two following headers are mutually exclusive in this sense.